Extensions/String+Extensions.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension String {
+    
+    var mergeUrl: String {
```

Comment:
The meaning isn't clear, should change to imagePath, url is URL,not String. if return type is String type, so should change to as Path. so imagePath should be fine